### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by RCopeland**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -98,6 +98,7 @@ module.exports = {
       {
         match: [
           './src/**/*.hbs',
+          '!./src/templates/drizzle/spark-core-icons.hbs',
```
Comment (afebbraro) [inline config.js]: yay!

File: src/pages/tests/icons.hbs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <svg class="sprk-c-Icon" viewBox="0 0 100 100">
+      <use xlink:href="#{{this.name}}" />
+    </svg>
+
+    <svg class="sprk-c-Icon sprk-c-Icon--large" viewBox="0 0 100 100">
+      <use xlink:href="#{{this.name}}" />
+    </svg>
+</div>
+{{else}}
+  Theres nothing
```
Comment (afebbraro) [inline src/pages/tests/icons.hbs]: haha :)

File: src/pages/tests/icons.hbs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+title: Icons
+layout: blank
+---
+<h1> Icons </h1>
+{{> drizzle.spark-core-icons}}
+
+
+{{#each (data "icons")}}
+<div class="drizzle-Space-inset-m">
+    <svg class="sprk-c-Icon sprk-c-Icon--small" viewBox="0 0 100 100">
```
Comment (afebbraro) [inline src/pages/tests/icons.hbs]: All the individual icons have different viewBoxes than the one they are getting from here. Are they displaying okay in your env? One option is to add a viewBox to the yml file and add `{{this.viewBox}}` into the viewbox attr like `viewBox="{{this.viewBox}}"`

Comment (RCopeland) [reply src/pages/tests/icons.hbs]: oh right, ill update that

**Pull request merged**